components/org.wso2.carbon.identity.remotefetch.common/src/main/java/org/wso2/carbon/identity/remotefetch/common/BasicRemoteFetchConfiguration.java

```diff
@@ -51,46 +52,85 @@ public BasicRemoteFetchConfiguration(int id, boolean isEnabled,
         this.failedDeployments = failedDeployments;
     }
 
+    /**
+     * get id.
+     * @return Id
+     */
     public int getId() {
 
         return id;
     }
 
+    /**
+     *get remoteFetchName.
+     * @return remoteFetName.
+     */
     public String getRemoteFetchName() {
 
         return remoteFetchName;
     }
 
+    /**
+     * get isEnabled which is used to point particular remoteFetchConfig is enabled or not.
+     * @return isEnabled.
+     */
     public boolean isEnabled() {
 
         return isEnabled;
     }
 
+    /**
+     * This string refers the type of remote repository manager eg : "GIT".
+     * {@link org.wso2.carbon.identity.remotefetch.common.repomanager.RepositoryManager}
+     * @return repositoryManagerType
+     */
     public String getRepositoryManagerType() {
 
         return repositoryManagerType;
     }
 
+    /**
+     * This string refers the type of actionListener eg: "POLLING".
+     * {@link org.wso2.carbon.identity.remotefetch.common.actionlistener.ActionListener}
+     * @return actionListenerType
+     */
     public String getActionListenerType() {
 
         return actionListenerType;
     }
 
+    /**
+     * This string refers the type of configurationDeployer eg : "SP".
+     * {@link org.wso2.carbon.identity.remotefetch.common.configdeployer.ConfigDeployer}
+     * @return configurationDeployerType
+     */
     public String getConfigurationDeployerType() {
 
         return configurationDeployerType;
     }
 
+    /**
+     * get number of successful deployments for list view.
+     * @return successfulDeployments
+     */
     public int getSuccessfulDeployments() {
 
         return successfulDeployments;
     }
 
+    /**
+     * get number of failed deployments for listview.
+     * @return failedDeployments
+     */
     public int getFailedDeployments() {
 
         return failedDeployments;
     }
 
+    /**
+     * get last deployed time for listview.
+     * @return lastDeployed
+     */
     public Date getLastDeployed() {
```

Comment:
Keep a space after the method name. Check other places as well.

Reply:
Thank you, This comment and your previous comments were fixed in
https://github.com/wso2-extensions/identity-fetch-remote/pull/40
https://github.com/wso2-extensions/identity-fetch-remote/pull/41
https://github.com/wso2-extensions/identity-fetch-remote/pull/42
https://github.com/wso2-extensions/identity-fetch-remote/pull/43
https://github.com/wso2-extensions/identity-fetch-remote/pull/44
https://github.com/wso2-extensions/identity-fetch-remote/pull/45.
I resolve this conversation.